zipline/assets/asset_writer.py

```diff
@@ -81,7 +81,7 @@
     'root_symbol': None,
     'asset_name': None,
     'start_date': 0,
-    'end_date': 2 ** 62 - 1,
+    'end_date': np.iinfo(np.int64).max,
```

Comment:
Ah. Nevermind. I just looked at your original comment in that we _used_ to have `2 ** 63 - 1`, which is just `np.int64`